### DISCUSSION TITLE
Any Typescript support or examples?

### DISCUSSION THREAD

**Discussion by VashJuan** [openpgpjs/openpgpjs#1564] [Ideas]
I'm completely new here and would love to see a Typescript example using OpenPgpJs. Are Typings available for this library? If so it might be worth a link and brief mention to help others with such a goal. I looked in the wiki and searched discussions but don't see this has been brought up at all.

Thanks for this impressive library!

**Comment by twiss** [comment]
Hey :wave: Yeah, there are TypeScript definitions, [here](https://github.com/openpgpjs/openpgpjs/blob/main/openpgp.d.ts). The closest thing we have to TypeScript-specific examples are the TS type definition tests, [here](https://github.com/openpgpjs/openpgpjs/blob/main/test/typescript/definitions.ts). But, generally speaking, the examples in the README should apply to TypeScript as well. If you see a situation where a TypeScript-specific example might be helpful, feel free to contribute it :)

**Reply by VashJuan** [reply]
Great. Thanks for the super fast (& helpful) reply.  If I get something working I'll see if I can post some code. Much appreciated.
